R/create_log_entry.R

```diff
@@ -0,0 +1,10 @@
+create_log_entry <- function(env_list) {
+  log_entry <- list()
+  log_entry$compile_init_env <- env_list$compile_init_env
+  log_entry$dots <- env_list$dots
+  log_entry$timestamp <- env_list$timestamp
+  log_entry$report_source_file <- env_list$report_source_file
+  log_entry$output_files <- env_list$output_files 
+  
+  return(log_entry)
```

Comment:
Maybe add a couple of comments here explaining why this function is useful, as it looks like it is returning its input. I guess the point here is that `env_list` can contain other items that we do not control, and we retain only the relevant ones?

Reply:
yep, you got it, will add that annotation